Fix musl compile since it does not define __sigemptyset

Downloaded from
http://git.alpinelinux.org/cgit/aports/commit/main/flac/sigemptyset.patch?id=49fd0f4cebc46e2753d6a60e450078446f7f18a7

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>

--- ./src/libFLAC/cpu.c.orig
+++ ./src/libFLAC/cpu.c
@@ -243,7 +243,7 @@
 		struct sigaction sigill_save;
 		struct sigaction sigill_sse;
 		sigill_sse.sa_sigaction = sigill_handler_sse_os;
-		__sigemptyset(&sigill_sse.sa_mask);
+		sigemptyset(&sigill_sse.sa_mask);
 		sigill_sse.sa_flags = SA_SIGINFO | SA_RESETHAND; /* SA_RESETHAND just in case our SIGILL return jump breaks, so we don't get stuck in a loop */
 		if(0 == sigaction(SIGILL, &sigill_sse, &sigill_save))
 		{
